proton

```diff
@@ -84,7 +84,10 @@ if not ("STEAM_COMPAT_DATA_PATH" in os.environ):
     log("No compat data path?")
     sys.exit(1)
 
-basedir = os.path.dirname(sys.argv[0])
+env = dict(os.environ)
+dlloverrides = {}
+
+basedir = env.get("WINE_BASEDIR", os.path.dirname(sys.argv[0]))
```

Comment:
you can just use `os.environ.get` here and leave the `env` and `dlloverrides` definition where they were

Reply:
True. But in my mind that is a tad redundant and begs the question why set the `os.environ` into it's own dict in the first place. 

I just was working within the code that was already set up by Valve and trying to avoid being opinionated. They're handling environment variables that way in the code, so maintain status quo unless another person wants to change it.

Reply:
they probably do so because they inject a bunch of variables into `env` that maybe they don't want to inject into the runtime environment, but they already do access `os.environ` further down in the code

Reply:
Forgot to respond. I see yeah. I did not read their code completely. I glanced at it and did this in about 5 minutes. If you're pretty sure that is the intention I don't mind going with the approach of using `os.environ` :+1: